app/workers/message_worker.rb

```diff
@@ -8,6 +8,7 @@ def self.enqueue(recipients, attributes)
   def perform(recipients, attributes)
     message = Message.new(attributes)
     message.sender_id = attributes["sender_id"]
+    message.subject = attributes["subject"]
```

Comment:
We still use the `protected_attributes` gem so it is possible that these attributes are protected and need individual assignment. I don't know if this is the case! Just saying.

But then if this **is** the case, to avoid warnings in log, we should filter them out when mass-assigning.

Reply:
yes true sender_id is the protected one and so assigned it this way and similar to that subject as well

Reply:
Ok, to avoid warnings in server log, you'd have to remove first these from `attributes` in the `::new` call.

Reply:
but is it really needed?

Reply:
Is it needed to avoid unnecessary warnings? I think yes. Reducing noise in logs helps catch actual things that need to be fixed.

Reply:
done

Reply:
@akostadinov Can we resolve this too?